htdocs/install/mysql/tables/llx_prelevement_facture_demande.sql

```diff
@@ -20,7 +20,10 @@
 create table llx_prelevement_facture_demande
 (
   rowid               integer AUTO_INCREMENT PRIMARY KEY,
+  entity              integer DEFAULT 1 NOT NULL,
   fk_facture          integer NOT NULL,
+  sourcetype          varchar(32),
+  fk_soc              integer NULL,
```

Comment:
Why adding fk_soc here ?
We already have fk_facture that allows to get invoice and so the fk_soc on invoice.

Reply:
@eldy payment intent works only with webhook and will concern all payment in dolibarr (from invoice but order, propal, subscription, donation, free payment....) so we need to save all informations for easy and quick insert with webhook 

all concerned object don't have fk_soc (ex:  pay with a free amount to a customer)

paymnt intent integration will be need for third quarter this year for all payment in europe

Reply:
But on this table, the field fk_facture is mandatory so the fk_soc can always be guess, so there is no need to store it.
I prefer to not duplicate information for the moment if we don't need it.
I may reconsider later once i will see that we really need it, but currently in this table this field seems a duplicate information that is not good for database integrity. Si i validate PR without it...